### PR TITLE
Expanding rtc acronym in selected cases

### PR DESCRIPTION
**Pull request by tylerkelly13**
Staged preview: http://docs-v5-feature-rtc-expansion.staging.tiny.cloud/docs/

Related Ticket: N/A

Description of Changes:
* Expanding RTC acronym to Real-Time Collaboration in selected cases to improve findability

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: plugins/premium/rtc.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ---
 layout: default
-title: RTC plugin
-title_nav: RTC (beta)
-description: Real time collaboration.
+title: Real-Time Collaboration plugin (beta)
+title_nav: Real-Time Collaboration (beta)
+description: The TinyMCE Real-Time Collaboration plugin
```
Comment (TheSpyder) [inline plugins/premium/rtc.md:5]: None of our other premium plugins have descriptions like this, starting with `The TinyMCE <blah>` 🤔  I do see a few `<blah> for TinyMCE`, though 🤔

**Review by TheSpyder — APPROVED** [review]

**Review by Afraithe — COMMENTED** [review]
Looks ok.

File: plugins/premium/rtc.md (hunk 1 above)
Comment (tylerkelly13) [reply plugins/premium/rtc.md:5]:
```suggestion
description: The Real-Time Collaboration plugin for TinyMCE
```
?

Comment (TheSpyder) [reply plugins/premium/rtc.md]: sure (I think I saw this in a couple of places, btw)

**Review by lnewson — APPROVED** [review]

**Review by JamesToohey — APPROVED** [review]

**Pull request merged**